Topics/Tech_Stacks/React.md

```diff
@@ -1,37 +1,59 @@
 
-### React Resources 
+To help ease the learning curve, below you'll find a variety of resources to reference while learning and components and libraries to use with React. 
 
-These frontend challenges for HTML, CSS and JavaScript will help you review core skills while also making interesting projects as well. \
-https://www.frontendmentor.io/
+### React Learning Resources 
 
+[Full-Stack Open](https://fullstackopen.com/)
+A full, free course provided by the University of Helsinki that teaches the MERN (MongoDB, Express, React, NodeJS) stack. It also covers TypeScript, GraphQL, React Native, and some CI/CD. An excellent resource for React beginners.
 
-Basic React tutorial provided by the official React website. This tutorial will guide you through a very basic tutorial that you can do straight from your browser. \
-https://reactjs.org/tutorial/tutorial.html
+[Coursera Learn React Basics](https://www.coursera.org/learn/react-basics)
+A free course provided by Coursera that covers the basics of React, including State, navigation, and taking user input via forms.
 
+[Free Scrimba React Course](https://scrimba.com/learn/learnreact)
+A free course provided by Scrimba that teaches React fundamentals through the creation of various applications including an AirBnB clone, a meme generator, and a notes app.
 
-React tutorial by MDN Web Docs. Useful readings for if you have time, but not necessary by any means.\
-https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/React_getting_started
+[Official React Tutorial](https://reactjs.org/tutorial/tutorial.html)
+Basic React tutorial provided by the official React website. This tutorial will guide you through a very basic tutorial that you can do straight from your browser. 
 
+[React JS Crash Course](https://www.youtube.com/watch?v=w7ejDZ8SWv8)
+[One Hour React Tutorial](https://www.youtube.com/watch?v=b9eMGE7QtTk)
+One hour in-depth tutorials about React. These Youtube crash course provides a great overview of React and various important components that you will be using often. Very useful if you are learning React under a strict time limit for whatever reason.
 
-Javascript frameworks tutorial provided by Microsoft. Useful readings, but not necessary to learn React. \
-https://learn.microsoft.com/en-us/windows/dev-environment/javascript/
+[Front-End Mentor](https://www.frontendmentor.io/)
+These frontend challenges for HTML, CSS and JavaScript will help you review core skills while also making interesting projects.
 
+[Mozilla React Tutorial](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/React_getting_started)
```

Comment:
MDN is excellent. I would strongly advocate adding the official React docs to this section, imo it is the best place for a beginner to learn React. But up to you

Reply:
true, added those as well.